tox.ini

```diff
@@ -15,11 +15,10 @@ commands=flake8 dag
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}
-deps =
-    -r{toxinidir}/requirements_dev.txt
+
+deps=.
```

Comment:
Can you please revert this change, we need to install requirements_dev over here for installing pytest, flake8 and other utilities

Reply:
Tell me what you think about the `requirements_test.txt` approach I presented 2 comments above (we could have both `-r{toxinidir}/requirements_test.txt` and `.` in tox for testing and project dependencies, respectively)